README.md

```diff
@@ -52,3 +52,9 @@ We love contributions! Check out the [Contribution guide](https://github.com/Kri
 ### Area#3: Natural Language Processing 
     - UseCase#1 : Credit Card Fraud Detection
     - UseCase#2 : Spam Or Ham classification 
+
+### Area#4: RoshanML-AI
```

Comment:
The area name should be generic, for example in your case you can create below three areas

- yoga-ai or you have it inside computer vision 
- stock-recommend-system
- essay-plagiarism-check 

Author name should not be part of the directory name.